src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,143 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_loginIsnull_NotOk() {
+        User user = new User();
+        user.setLogin(null);
+        user.setPassword("validPassword");
+        user.setAge(23);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void chec_login_length_0_notOk() {
+        User user = new User();
+        user.setLogin("");
+        user.setPassword("validPassword");
+        user.setAge(23);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_loginlengthLessThanRequired_notOk() {
+        User user = new User();
+        user.setLogin("euryt");
+        user.setPassword("validPassword");
+        user.setAge(23);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void chec_login_exist_notOk() {
+        User user = new User();
+        user.setLogin("kolner");
+        user.setPassword("kolnerevgen");
+        user.setAge(30);
+        Storage.PEOPLE.add(user);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void chec_login_length_6_Ok() {
```

Comment:
same

Reply:
and does not use a specific value in the name, because if this value changes from 6 to 10, then there will be a problem. Same everywhere